io.catenax.product_stock/1.0.0/ProductStock.ttl

```diff
@@ -0,0 +1,143 @@
+:AllocatedStock a bamm:Entity;
+    bamm:properties (:quantityOnAllocatedStock :supplierStockLocationId);
+    bamm:preferredName "Stock allocated to Customer"@en;
+    bamm:description "This is the quantity of already produced goods at a given location that has not yet been shipped. The allocated stock is the stock available for the customer and may refer to a given position (see OrderPositionReference). Quantities of Materials belonging to this stock are not available for other positions or customers. This stock consists only of the good parts."@en.
+:supplierOrderId a bamm:Property;
+    bamm:preferredName "Supplier Order ID"@en;
+    bamm:description "This is the order number on the supplier side that is given by e.g., the supplier's enterprise resource planning system."@en;
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "M-Nbr-4711".
+:customerOrderId a bamm:Property;
+    bamm:preferredName "Customer Order ID"@en;
+    bamm:description "This is the order number on the customer side that can be given by e.g., the customer's enterprise resource planning system."@en;
+    bamm:characteristic :OrderIdCharacteristic;
+    bamm:exampleValue "C-Nbr-4711".
+:customerOrderPositionId a bamm:Property;
+    bamm:preferredName "Customer Order Position ID"@en;
+    bamm:description "This is an identifier for the position of an order defined by the customer. It's not a technical identifier, such as an UUID. This identifier is used within the business context when talking about the position."@en;
+    bamm:characteristic :OrderPositionIdCharacteristic;
+    bamm:exampleValue "PositionId-01".
+:quantityOnAllocatedStock a bamm:Property;
+    bamm:preferredName "Quantity on Allocated Stock"@en;
+    bamm:description "This is the quantity of the material within the position that is already ready to be delivered."@en;
+    bamm:characteristic ext-planned:QuantityCharacteristic.
+:OrderIdCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Order ID Characteristic"@en;
+    bamm:description "This is a proprietary order ID, generated by either the supplier side or customer side."@en;
+    bamm:dataType xsd:string.
+:OrderPositionIdCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Position ID within Order"@en;
+    bamm:description "The position ID is unique for an order and used within business context to refer to a position."@en;
+    bamm:dataType xsd:string.
+:LocationKeyTypeEnum a bamm-c:Enumeration;
+    bamm:preferredName "Location Key Type Enum"@en;
+    bamm:description "Enumeration specifying types of keys for locations within Catena-X."@en;
+    bamm:dataType xsd:string;
+    bamm:see <urn:uri:CX-0010>;
+    bamm-c:values ("BPNS" "BPNA").
+:LocationKeyValueCharacteristic a bamm:Characteristic;
+    bamm:preferredName "Value for specific Location Key Characteristic"@en;
+    bamm:description "The actual key based on the type."@en;
+    bamm:dataType xsd:string;
+    bamm:see <urn:uri:CX-0010>.
```

Comment:
same here. please check all the see elements and change to a proper uri (eg URL)